package.json

```diff
@@ -8,7 +8,7 @@
     "@testing-library/user-event": "^13.5.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
-    "react-scripts": "5.0.1",
+    "react-scripts": "^5.0.1",
```

Comment:
in real life projects, be careful with the package versions, if not necessary do not update them, do not include them in your PR